### ISSUE TITLE
How can I use yolo to identify small objects that match the background color?

### ISSUE THREAD

**Issue by bobo970722** [ultralytics/ultralytics#18489]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello，
I want to identify a small black object against a black background on the back of the phone, this object is a circle of glue around the phone, I have 5000 datasets, can I use yolo to identify it?

Thank you so much,
Bobo

**Comment by UltralyticsAssistant** [comment]
👋 Hello @bobo970722, thank you for your interest in Ultralytics 🚀! Your question brings up an intriguing use case. It is definitely possible to use YOLO for identifying small objects that share the same color as the background, though it may require careful dataset preparation and parameter finetuning.

To help assist further, if this is a 🐛 Bug Report, please provide a minimum reproducible example, including the code, datasets, and exact steps to replicate your results so we can better understand the issue.

If this is a custom training ❓ Question like it seems to be, kindly share additional details such as sample images from your dataset, the format of your labels, training parameters (batch size, epochs, image size, etc.), and any relevant training logs. Also, when working with challenging objects, you may find it helpful to follow tips for achieving the best training results, such as adjusting your training strategy, experimenting with different augmentation techniques, or increasing the resolution.

Additionally, you can always get help from the community! For real-time discussion, you can join our Discord server 🎧, participate in deeper forum discussions on Discourse, or share insights on our Subreddit.

For now, please ensure you're using the latest version of Ultralytics and all dependencies. You can upgrade by running:

```bash
pip install -U ultralytics
```

Lastly, make sure you’re running your experiments in a correct and verified environment, such as local Python, Notebook-based setups, or cloud platforms like Gradient or Colab. An Ultralytics engineer will also assist soon to provide further guidance. Good luck, and happy experimenting! 😊

**Comment by glenn-jocher** [comment]
@bobo970722 yes, you can use YOLO to identify small objects like the black glue circle on a black background. To achieve the best results, ensure high-quality labeling of your dataset and consider fine-tuning a pre-trained YOLO model, such as YOLO11, on your dataset. Additionally, augmentations like contrast adjustments or lighting enhancements may help highlight the object during training. For further guidance, refer to the training documentation: https://docs.ultralytics.com/modes/train.
